src/test/java/com/artipie/FilesRepoITCase.java

```diff
@@ -168,9 +154,20 @@ void tearDown() {
         this.cntn.stop();
     }
 
-    private String exec(final String... command) throws Exception {
-        Logger.debug(this, "Command:\n%s", String.join(" ", command));
-        return this.cntn.execInContainer(command).getStdout();
+    private String curl(final boolean anonymous, final String action,
+        final ArtipieServer.User usr) throws Exception {
```

Comment:
@genryxy I'd suggest to refactor the method so it accepts `Optional<ArtipieServer.User>` instead of ArtipieServer.User` and remove `anonymous` flag. This way we avoid input that makes no sense with `anonymous=true` and some user specified